l10n.js

```diff
@@ -219,40 +223,70 @@ document.webL10n = (function(window, document, undefined) {
               parsedRawLinesCallback();
               return;
             }
-            var line = entries.shift();
```

Comment:
I've added a check for escaped backslashes at the end and am re-jigging some of this code now.